pyccl/haloprofile.py

```diff
@@ -24,8 +24,7 @@ def nfw_profile_3d(cosmo, concentration, halo_mass, odelta, a, r):
         float or array_like: 3D NFW density at r, in units of Msun/Mpc^3.
     """
     mdef = hal.MassDef(odelta, 'matter')
-    c = hal.ConcentrationConstant(c=concentration,
-                                  mdef=mdef)
+    c = hal.ConcentrationConstant(c=concentration, mdef=mdef)
```

Comment:
I thought we had agreed on homogenising to `mass_def`

Reply:
fixed